uk.co.compucorp.civicrm.tasksassignments/tests/phpunit/CRM/Tasksassignments/BAO/DocumentTest.php

```diff
@@ -50,18 +50,6 @@ function testCreateDocumentWithNoSource() {
   /**
    * @expectedException CRM_Exception
    */
-  function testCreateDocumentWithNoAssignee() {
```

Comment:
removing expectException is not enough , you still need to do some assertion on the result  ..  I will run them on my machine once you fix this